include/megaapi.h

```diff
@@ -6729,6 +6732,203 @@ class MegaApi
          */
         bool isOnline();
 
+#ifdef HAVE_LIBUV
+        /**
+         * @brief Start an HTTP proxy server in specified port
+         *
+         * If this function returns true, that means that the server is
+         * ready to accept connections. The initialization is synchronous.
+         *
+         * The server will serve files using this URL format:
+         * http://127.0.0.1/<NodeHandle>/<NodeName>
+         *
+         * The node name must be URL encoded and must match with the node handle.
+         * You can generate a correct link for a MegaNode using MegaApi::httpServerGetLocalLink
+         *
+         * If the node handle belongs to a folder node, a web with the list of files
+         * inside the folder is returned.
+         *
+         * @param localOnly true to listen on 127.0.0.1 only, false to listen on all network interfaces
+         * @param port Port in which the server must accept connections
+         * @return True is the server is ready, false if the initialization failed
+         */
+        bool httpServerStart(bool localOnly = true, int port = 4443);
+
+        /**
+         * @brief Stop the HTTP proxy server
+         *
+         * When this function returns, the server is already shutdown.
+         * If the HTTP proxy server isn't running, this functions doesn't do anything.
+         */
+        void httpServerStop();
+
+        /**
+         * @brief Check if the HTTP proxy server is running
+         * @return 0 if the server is not running, otherwise the port in which it's listening
```

Comment:
@return 0 if the server is not running. Otherwise, the port in which it's listening to